scripts/hudson/narayana.sh

```diff
@@ -927,7 +896,7 @@ function perf_tests {
 
   comment_on_pull "$PERF_OUTPUT" "$BUILD_URL"
 
-  [ $res = 0 ] || fatal "there were regressions in one or more of the benchmarks (see previous PR comment for details"
+  [ $res = 0 ] || fatal "there were regressions in one or more of the benchmarks (see previous PR comment for details)"
```

Comment:
I think this tidy up could go in a separate commit

Reply:
lol, ok